task/rsync/0.1/README.md

```diff
@@ -0,0 +1,237 @@
+# Rsync
+
+This task provides synchronization of files between container and remote host. It relies on rsync and sshpass commands to implement.
```

Comment:
it's maybe not the most secure way to do rsync, what about ssh key exchange instead?

a username/password leaking is worse than a key that can be confined in authorized_keys

Reply:
I will remove username/password and replace it by authorized_keys.

Reply:
I have added ssh authentication.